### ISSUE TITLE
GU0011 false negatives

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#154]
```cs
        internal static DocumentEditor AddUsing(this DocumentEditor editor, UsingDirectiveSyntax usingDirective)
        {
            editor.ReplaceNode(
                editor.OriginalRoot,
                (root, _) => AddUsing(root as CompilationUnitSyntax, editor.SemanticModel, usingDirective));

            return editor;
        }
```
![animation](https://user-images.githubusercontent.com/1640096/37758375-452bac84-2db0-11e8-9929-3fe74224c31b.gif)

The warning goes away when navigating to it, very annoying.
